HW0/P4/P4.py

```diff
@@ -20,6 +20,10 @@
     rx = 1.0
     ry = 5.0
     rz = 5.0
+    C = np.zeros((3,3))
+    C[0,0] = rx
```

Comment:
Shouldn't this and all the ones following be reciprocals (ie 1/rx)? At least that's what I gathered from the HW spec on page 17/22.